### PR TITLE
fix: use correct event manager for actor methods

### PR DESCRIPTION
**Pull request by vladfrangu**
This fixes the actor side of https://github.com/apify/crawlee/issues/1463

For the crawlee only side, I'll open up a PR there

### CODE REVIEW

File: packages/apify/src/actor.ts [hunk 1]
```diff
@@ -179,6 +177,9 @@ export class Actor<Data extends Dictionary = Dictionary> {
             this.config.useStorageClient(options.storage);
         }
 
+        // Init the event manager the config uses
```
Comment (B4nan) [inline packages/apify/src/actor.ts:180]: this looks wrong, we need to use the `PlatformEventManager`, without it the platform events (from websockets) wont work at all.

Comment (vladfrangu) [reply packages/apify/src/actor.ts:180]: The if check above will make the config use the PlatformEventManager, no?

Comment (B4nan) [reply packages/apify/src/actor.ts:180]: Right, I am blind. But then I dont understand why did you have to change it to use the one from config?

Comment (B4nan) [reply packages/apify/src/actor.ts:180]: Or is the fix just about moving the `init` call and the rest is unrelated? i mean the changes from `this.eventManager` to `this.config.getEventManager()`

Comment (B4nan) [reply packages/apify/src/actor.ts:180]: Oh wait a sec, now I get it. We dont want to use the platform implementation locally, but we did.

Comment (vladfrangu) [reply packages/apify/src/actor.ts:180]: Yep, thats the reason!

**Review by B4nan — APPROVED** [review]

**Review by B4nan — APPROVED** [review]

**Pull request merged**